### ISSUE TITLE
Method verifySelectOptions KO in VBA / Selenium Wrapper

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#117]
```
Operating system and version (32/64bit) : Windows 7 64bits
Office name and version(32/64bit) : Excel 2013
Browser name and version : Mozilla FF 31.0
SeleniumWrapper version : 1.0.21.1

It is not possible to use this method in a VBA code. I have an error when I try 
to execute this method.

You can see the attached file.
```

Original issue reported on code.google.com by `dnowak...@gmail.com` on 18 Nov 2014 at 8:56

Attachments:
- [2014-11-18_095505.png](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-117/comment-0/2014-11-18_095505.png)

**Comment by GoogleCodeExporter** [comment]
```
This error is telling that the method was not found for the provided arguments.
If you check the definition of the method, you'll see that the method needs a 
string and array for arguments.
You could use Split to convert the values in a cell to an array:
Debug.Print wd.verifySelectOptions("locator", Split(Range("A1"), ";"))

```

Original comment by `florentbr` on 18 Nov 2014 at 1:11
- Changed state: **Accepted**
- Added labels: **Type-Support**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for your answer ^^
I tried that :

1- I have the value "Yes;No;Maybe" on Excel (Range "D16").
2- I try to execute the following VB code :

Dim selenium As New SeleniumWrapper.WebDriver
ControlResult = selenium.verifySelectOptions("id=selectedCivility", 
Split(Range("D16"), ";"))

I have the same error (see new attached file).

Thks for support !


```

Original comment by `dnowak...@gmail.com` on 18 Nov 2014 at 2:06

Attachments:
- [2014-11-18_150545.png](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-117/comment-2/2014-11-18_150545.png)

**Comment by GoogleCodeExporter** [comment]
```
I managed to reproduce it and it's a bug.
It will be fixed before the end of the week in the next release(1.0.23)
```

Original comment by `florentbr` on 18 Nov 2014 at 5:39
- Added labels: **Type-Defect**
- Removed labels: **Type-Support**

**Comment by GoogleCodeExporter** [comment]
```
OK : thanks for your action !
```

Original comment by `dnowak...@gmail.com` on 18 Nov 2014 at 10:06

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.0.23
```

Original comment by `florentbr` on 20 Nov 2014 at 9:42
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Great : it's OK for me with this new 1.0.23 release ^^
Thks !
```

Original comment by `dnowak...@gmail.com` on 20 Nov 2014 at 10:14
